docker-compose.build.yml

```diff
@@ -12,7 +12,7 @@ services:
   ########################################################################
 
   sysmon-generator:
-    image: ${DOCKER_REGISTRY:-registry-1.docker.io}/grapl/sysmon-generator:${TAG:-latest}
+    image: ${DOCKER_REGISTRY:-docker.io}/grapl/sysmon-generator:${TAG:-latest}
```

Comment:
Not to flog a dead horse, but this could be simplified if we require that `$DOCKER_REGISTRY` ends in a `/`.

Then you could just use `${DOCKER_REGISTRY:-}grapl/sysmon-generator:${TAG:-latest}`... if you supply a registry, it gets added, but if not, you still have a valid identifier.

Not required or anything, though; just something that occurred to me just now.